bot.py

```diff
@@ -4,37 +4,31 @@
 import discord
 from discord.ext import commands
 from dotenv import load_dotenv
-GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
 
+load_dotenv()
+
+GITHUB_TOKEN = os.getenv('GITHUB_TOKEN')
 g = Github(GITHUB_TOKEN)
 
-load_dotenv()
 DISCORD_TOKEN = os.getenv('DISCORD_TOKEN')
-bot = commands.Bot(command_prefix='!')
+bot = commands.Bot(command_prefix='!git ')
 
 # get current user
-user = g.get_user("Laurell876")
-print(user.name)
-#repo = g.get_repo("MartinHeinz/python-project-blueprint")
-
-#for repo in g.get_user().get_repos():
-#    print(repo.name)
-
-@bot.command(name='git')
-async def test(ctx):
-	response = 'hey dirtbag'
-	await ctx.send(response)
-
-#discord.Client()
-
-'''
-@bot.event
-async def on_ready():
-    print(f'{bot.user} has connected to Discord!')
-
-@bot.event
-async def on_message(message):
-	if message.content == "hello":
-		await message.channel.send("hey dirtbag")
-'''
+
+user = g.get_user()
+print(user.name) 
+
+for repo in user.get_repos():
+    print(repo.name)
+
+
+@bot.command()
+async def hello(ctx): #!git hello
+	await ctx.send('Hi there '+ctx.message.author.name)
+
+@bot.command()
+async def create_repo(ctx, repoName): #!git create_repo repo1
+	repo = user.create_repo(repoName)
```

Comment:
Why are you getting the `repo` return value here? If you're interested in using it later maybe it'd be better to return that after sending the "repo created" message